aana/exceptions/io.py

```diff
@@ -102,3 +102,30 @@ class VideoReadingException(VideoException):
     """
 
     pass
+
+
+class VideoTooLongException(BaseException):
+    """Exception raised when the video is too long.
+
+    Attributes:
+        video (Video): the video that caused the exception
+        video_len (float): the length of the video in seconds
+        max_len (float): the maximum allowed length of the video in seconds
+    """
+
+    def __init__(self, video: "Video", video_len: float, max_len: float):
+        """Initialize the exception.
+
+        Args:
+            video (Video): the video that caused the exception
+            video_len (float): the length of the video in seconds
+            max_len (float): the maximum allowed length of the video in seconds
```

Comment:
DO we need this twice?

Reply:
Do you mean Attributes and Args? Not sure but I keep it consistent with the exceptions we had before.